app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -21,6 +21,8 @@ def user_info
     info.merge!(ial2_attributes) if scoper.ial2_scopes_requested?
     info.merge!(x509_attributes) if scoper.x509_scopes_requested?
     info[:verified_at] = verified_at if scoper.verified_at_requested?
+    info[:ial] = ial_value(identity.ial) if identity.ial.present?
```

Comment:
```suggestion
    info[:ial] = Saml::Idp::Constants::AUTHN_CONTEXT_IAL_TO_CLASSREF[identity.ial] if identity.ial.present?
```

https://github.com/18F/identity-idp/blob/7622467ec0482e4443431ca1d3ab0f19aa81a59d/lib/saml_idp_constants.rb#L46 should be able to take care of this translation without the method